src/main/java/org/folio/entlinks/config/constants/ErrorCode.java

```diff
@@ -3,6 +3,7 @@
 import lombok.Getter;
 
 public enum ErrorCode {
+  UNKNOWN_CONSTRAIN("100", "Unknown constrain."),
```

Comment:
```suggestion
  UNKNOWN_CONSTRAINT("-1", "Unknown constraint."),
```
It's okay to have such a default value, but I think we should put `-1` as the error code to indicate that it is an unexpected exception that is out of scope of current implemntation.

Reply:
fixed